### ISSUE TITLE
const keyword ignored when falsey

### ISSUE THREAD

**Issue by Leftium** [StefanTerdell/json-schema-to-zod#30]
Given this JSON Schema:
```
{
  type: "object",
  properties: {
    s: {
      type: "string",
      const: ""
    }
  }
}
```
This result is expected (`z.literal("")`):
```
import { z } from "zod";

export default z.object({ s: z.literal("").optional() });
```
However the actual result is (`z.string()`):
```
import { z } from "zod";

export default z.object({ s: z.string().optional() });
```

---

It seems `json-schema-to-zod` outputs the incorrect result because the string literal "" is falsey. It seems this is a problem for all [falsey values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).

**Comment by StefanTerdell** [comment]
@Leftium I'm unable to reproduce this. Could you check out the cases added in #31 ?

**Comment by StefanTerdell** [comment]
@Leftium Tested the wrong issue 😅 A fix is out in 0.6.1
